src/modules/blocks/event-datetime/controls/template.js

```diff
@@ -29,65 +29,63 @@ import {
 
 const { timezonesAsSelectData } = date;
 
-class EventDateTimeControls extends PureComponent {
-	static propTypes = {
-		isEditable: PropTypes.bool.isRequired,
-		onTimeZoneVisibilityChange: PropTypes.func,
-		separatorDate: PropTypes.string,
-		separatorTime: PropTypes.string,
-		setSeparatorDate: PropTypes.func,
-		setSeparatorTime: PropTypes.func,
-		setTimeZone: PropTypes.func,
-		showTimeZone: PropTypes.bool,
-		timeZone: PropTypes.string,
-	};
+const EventDateTimeControls = ( props ) => {
+	const {
+		separatorDate,
+		separatorTime,
+		setSeparatorDate,
+		setSeparatorTime,
+		setShowTimeZone,
+		setTimeZone,
+		showTimeZone,
+		timeZone,
+		isEditable,
+	} = props;
 
-	render() {
-		const {
-			onTimeZoneVisibilityChange,
-			separatorDate,
-			separatorTime,
-			setSeparatorDate,
-			setSeparatorTime,
-			setTimeZone,
-			showTimeZone,
-			timeZone,
-			isEditable,
-		} = this.props;
+	return isEditable && (
```

Comment:
I think we'd have to be super sure that `isEditable` is required here. If not, I believe it would error out saying something like nothing is return from the render. I usually err on the side of caution by using a conditional ternary operator and return null as fallback.

```suggestion
	return isEditable ? ( ... ) : null
```

Reply:
`isEditable` will always be passed here, it's initialized before the blocks are loaded. If it is `false`, then it renders `false`, which is nothing. If `true`, then it'll render the template.